mappings/net/minecraft/fluid/BaseFluid.mapping

```diff
@@ -43,5 +44,32 @@ CLASS net/minecraft/class_3609 net/minecraft/fluid/BaseFluid
 		ARG 4 newState
 	METHOD method_17774 isFluidAboveEqual (Lnet/minecraft/class_3610;Lnet/minecraft/class_1922;Lnet/minecraft/class_2338;)Z
 		ARG 0 state
-		ARG 1 view
+		ARG 1 world
+		ARG 2 pos
+	METHOD method_15726 getSpread (Lnet/minecraft/class_4538;Lnet/minecraft/class_2338;Lnet/minecraft/class_2680;)Ljava/util/Map;
```

Comment:
Apparently this class doesn't exist: `Lnet/minecraft/class_4538;`

Reply:
Lemme double check this, github is showing me outdated classes...

Reply:
`getSpread` got removed in the latest commit with Enigma's drop invalid mappings.